### ISSUE TITLE
Incompatible NVDA Version Keeps Add-ons Disabled in Compatible Versions

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#15412]
### Steps to reproduce:
1. Upgrading from 2023.2 to the latest alpha-29134 There are some installed add-ons that are disabled because they are incompatible with 2024.1.
2. Downgrade NVDA back to 2023.2.
### Actual behavior:
Add-ons that are disabled because they are incompatible with the NVDA version that breaks compatibility are still disabled in compatible NVDA versions
### Expected behavior:
These add-ons should be available again in NVDA2023.2 as they are actually compatible with NVDA-2023.2.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA-alpha-29134, and NVDA-2023.2
#### Windows version:
Windows 10 22H2 (AMD64) build 19045.3393
#### Name and version of other software in use when reproducing the issue:
none

#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Not tested
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by CyrilleB79** [comment]
I have encountered the same issue because I have been using the same config folder with the installer of the alpha (using "-c %appdata%\nvda" command line option). Worse: when turning back to installed NVDA 2023.2, not only are all the add-ons disabled, but it's even impossible to re-enable them.

Note however that NVDA downgrade is not a supported use case.

@seanbudd, if a simple change can be done anyway for this, that would be a nice to have; but I understand that NV Access will not spend some time to support a downgrade scenario.

**Comment by seanbudd** [comment]
Closing this as we don't support downgrading.
